ui/components/multichain/pages/send/send.js

```diff
@@ -248,6 +248,9 @@ export const SendPage = () => {
     [dispatch],
   );
 
+  const isSwapAndSend =
+    transactionAsset?.details?.address !== receiveAsset?.details?.address;
+
```

Comment:
Minor: can be extracted to a util or selector

Reply:
Done